tests/nix/simple/default.nix

```diff
@@ -11,8 +11,6 @@ in
 pkgs.mkShell {
   buildInputs = with pkgs; [
     python36Packages.numpy
-    python36Packages.scipy
-    python36Packages.jupyterlab
```

Comment:
`python36Packages.jupyterlab` really should be mandatory. Otherwise `jupyterlab` will not be installed in the nix environment.

Reply:
Nope. A test can do what it wants really because the command that is executed in the container are those listed in the `verify` script.

Was trying to cut down on the things that we install in each test to gain time as the sum of all our tests takes the age of the universe to execute :( so I am currently trying to remove things from them when I pass by.